apps/prairielearn/elements/pl-order-blocks/pl-order-blocks.py

```diff
@@ -285,13 +285,7 @@ def prepare_tag(
             )
 
         if format is FormatType.CODE:
-            inner_html = (
-                "<pl-code"
-                + (' language="' + code_language + '"' if code_language else "")
-                + ">"
-                + inner_html
-                + "</pl-code>"
-            )
+            inner_html = f'<pl-code language="{code_language or ""}">{inner_html}</pl-code>'
```

Comment:
Upon further review: this is slightly a change in behaviour. Previously, if code_language was unset, this would skip the `language="..."` argument, while now it sets it to blank. I just tested, and this causes `pl-code` to try to find the language "", which does not exist.

Reply:
my bad, should be fixed now